packages/lesswrong/components/posts/dialogues/DialogueSubmit.tsx

```diff
@@ -107,6 +118,33 @@ const DialogueSubmit = ({
         >
           {saveDraftLabel}
         </Button>
+        {userShortfeed && <Button
+          className={classNames(classes.formButton)}
+          disabled={loading || !!error}
+          onClick={async e => {
+            e.preventDefault()
+            
+            // So this property exists. But the typings don't agree. For now, #AnyBecauseHard
+            // @ts-ignore
+            const shortformString = editors[`${document._id}-contents`].getData()
```

Comment:
What's the complaint Typescript has here?  If `editors` is a `Record<string, Editor>` then I don't see why it'd have any issues

Reply:
doesn't believe in `.getData()`. Will disambiguate what "this property" means

Reply:
Oh, huh.  Makes me wonder if the type is wrong, in that case.